src/DBConnector/Connector.java

```diff
@@ -44,25 +71,36 @@ public Connector() {
   /**
    * print all the current usernames
    */
-  public void printAllUsers() {
-
-    String sql = "select username from " + tableName;
-
+  public String[] getAllUsers() {
+    ResultSet result = null;
+    ResultSet resultSize = null;
     try {
-
-      ResultSet result = myStmt.executeQuery(sql);
+      resultSize = selectCount.executeQuery();
+      resultSize.next();
+      int size = resultSize.getInt(1);
+      String[] userNames = new String[size];
```

Comment:
`List<String> userNames = new ArrayList<>();`
I wouldn't worry about setting capacity on your list at this point.